packages/features/shell/Shell.tsx

```diff
@@ -780,7 +780,11 @@ export function ShellMain(props: LayoutProps) {
 
   return (
     <>
-      <div className={classNames("mb-4 flex sm:mt-0 md:mb-6", props.smallHeading ? "lg:mb-7" : "lg:mb-8")}>
+      <div
+        className={classNames(
+          "mb-4 flex items-center sm:mt-0 md:mb-6",
+          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
+        )}>
```

Comment:
I'd recommend you to commit a simple change, which will also solve #7290 
```suggestion
        className={classNames(
          "my-4 flex items-center md:mt-0 md:mb-6",
          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
        )}>
```

Reply:
Pushed it, thanks!